<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TabPrincipale">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="TabRechercher">
       <attribute name="title">
        <string>Rechercher</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Recherche">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>951</width>
          <height>531</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>No. article</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pièce</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fournisseur</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantité en stock</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date d'aquisition</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date dernière vente</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prix de vente</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Emplacement</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Chemin image</string>
         </property>
        </column>
       </widget>
       <widget class="QComboBox" name="comboBox_Recherche">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>31</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="TabAjouter">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="TabModifier">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_Recherche</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
